howso/marginal.amlg

```diff
@@ -489,7 +469,7 @@
 								;true means there are nulls, false means no nulls, null means unknown whether there are nulls
 								"has_nulls" (> num_nulls 0)
 								"null_residual"
-									(if (!= .nan (- max min))
+									(if (!= (null) (- max min))
```

Comment:
should be able to replace this `if` block with one liner `or` like so:  `(or (- max min) 0)`

Reply:
I'm gonna keep it this way as it's readable